Helper/ClientManager.php

```diff
@@ -120,4 +120,10 @@ private function getSetupVersion()
             : "";
     }
 
+    public function getMagentoVersion()
```

Comment:
Shouldn't ClientManager be related only to getting an instance of the Sailthru client? I feel like this method makes more sense elsewhere. For example, in Settings; that feels more generic to me.

Reply:
moved